src/cpp/utils/SystemInfo.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include "SystemInfo.hpp"
 
+#include <sys/stat.h>
```

Comment:
How portable is this? (e.g., Windows). I'm relaunching tests of windows because they failed for other reasons.

Reply:
It seems to work, so OK :+1